package.json

```diff
@@ -36,6 +36,8 @@
     "test": "npx jest",
     "test:integration": "./scripts/test-integration",
     "test:watch": "jest --watch",
```

Comment:
this is probably an unnecessary command here, for the same reason as my comment below

Reply:
Will remove both 👍